platform/notifications/core/platform-core-notificationcommands/src/main/java/org/codice/ddf/notifications/commands/SendCommand.java

```diff
@@ -110,7 +111,7 @@ public Object execute() throws Exception {
       sendNotification();
       if (waitTime > 0) {
         try {
-          Thread.sleep(waitTime * 1000);
+          Thread.sleep(waitTime * 1000L);
         } catch (InterruptedException e) {
 
```

Comment:
❗️ Interruption should be properly propagated and not just swallowed. Typically this is done by setting a boolean variable to true and outside the loop, put a try/finally that checks for this flag to be true and propagates the interruption exception to the current thread.